dyno/microcontroller/Session.ino

```diff
@@ -20,16 +24,36 @@ void Session::init_temp_sense(const int *TEMP_PINS) {
 }
 
 void Session::process_command() {
-  if (com_buff_count == 1) {
+  if (m_com_buff_count == 1) {
```

Comment:
Will always send a flag with a semi-colon at the end. Is that handled on line 66? Or do we need to handle the semicolon here?

Reply:
It should be handled on line 66. The idea is that, within the scope of `process_command()`, you have exactly 1 full command to parse and execute. So in this context, `m_com_buff_count` contains the number of characters in the command itself, excluding the semicolon.